content/en/docs/reference/labels-annotations-taints/_index.md

```diff
@@ -431,6 +431,14 @@ Used on: PersistentVolumeClaim
 
 This annotation has been deprecated.
 
+### volume.beta.kubernetes.io/storage-class (deprecated)
+
+Used on: PersistentVolume, PersistentVolumeClaim
+
+This annotation can be used for PersistentVolume(PV) or PersistentVolumeClaim(PVC) to specify the name of [StorageClass](/docs/concepts/storage/storage-classes/). When both `storageClassName` attribute and `volume.beta.kubernetes.io/storage-class` annotation are specified, the annotation `volume.beta.kubernetes.io/storage-class` takes precedence over the `storageClassName` attribute.
+
+This annotation has been deprecated.
```

Comment:
```suggestion
This annotation has been deprecated. Instead, set the [`storageClassName` field](/docs/concepts/storage/persistent-volumes/#class)
for the PersistentVolumeClaim or PersistentVolume.
```

Reply:
That was what I wanted to suggest. For any deprecated annotation, there must be something else as an alternative.

Reply:
@sftim @tengqm Thanks once again for taking time to discuss this and review the changes. I have incorporated all the suggested feedback with new commits.